runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaPipelineOptions.java

```diff
@@ -111,6 +111,12 @@ public interface SamzaPipelineOptions extends PipelineOptions {
 
   void setEnableMetrics(Boolean enableMetrics);
 
+  @Description("Enable/disable Beam Transform throughput, latency metrics in Samza Runner")
+  @Default.Boolean(true)
```

Comment:
Let's keep default to be false so we can enable it only for jobs need this.

Reply:
done